lighthouse-core/gather/gatherers/iframe-elements.js

```diff
@@ -18,12 +20,11 @@ function collectIFrameElements() {
   // @ts-expect-error - put into scope via stringification
   const iFrameElements = getElementsInDocument('iframe'); // eslint-disable-line no-undef
   return iFrameElements.map(/** @param {HTMLIFrameElement} node */ (node) => {
-    const clientRect = node.getBoundingClientRect();
-    const {top, bottom, left, right, width, height} = clientRect;
     return {
+      // @ts-expect-error - getNodeInfo put into scope via stringification
```

Comment:
we might need to be careful about these expect errors when spread into the object, I think it might be masking the type error from the missing `clientRect`?